### PR TITLE
Update UPGRADE.md

### PR DESCRIPTION
**Pull request by sakshamtaneja21**
### Summary

SUMMARY_GOES_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #XXX

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -10,12 +10,12 @@ Unless indicated otherwise in one of the upgrade paths of this document, it is
 possible to upgrade Kong **without downtime**:
 
 Assuming that Kong is already running on your system, acquire the latest
-version from any of the available [installation
+the version from any of the available [installation
```
Comment (levfishbluefish) [inline UPGRADE.md]: this is a continuation of the previous line "acquire the latest version"

File: UPGRADE.md [hunk 2]
```diff
@@ -10,12 +10,12 @@ Unless indicated otherwise in one of the upgrade paths of this document, it is
...
 methods](https://getkong.org/install/) and proceed to install it, overriding
 your previous installation.
 
 If you are planning to make modifications to your configuration, this is a
-good time to do so.
+A good time to do so.
```
Comment (levfishbluefish) [inline UPGRADE.md]: incorrect change, this is a continuation of the previous line "this is a good time to do so"

File: UPGRADE.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ configuration, without dropping existing in-flight connections.
 
 Kong adheres to [semantic versioning](https://semver.org/), which makes a
 distinction between "major", "minor" and "patch" versions. The upgrade path
-will be different on which previous version from which you are migrating.
+will be different on which the previous version from which you are migrating.
```
Comment (levfishbluefish) [inline UPGRADE.md]:
```suggestion
will be different depending on the previous version from which you are migrating.
```

**Comment by sakshamtaneja21** [comment]
done

**Pull request closed without merging**